File: Modul-02/C/PRAK205-2310817210009-Muhammad-Ibnu-Sina.c
#include <stdio.h>
#include <math.h>

int main(){

    int TINGGI, MIRING;

    printf("Input: \n");
    scanf("%d", &TINGGI);
    scanf("%d", &MIRING);

    int alas = sqrt(pow(MIRING,2)-(pow(TINGGI,2)));
    int tinggi = TINGGI;
    int keliling = TINGGI + MIRING + alas;
    int luas = 0.5 * alas * tinggi;

    printf("\nOutput: \nAlas = %d cm\n", alas);
    printf("tinggi = %d cm\n", tinggi);
    printf("keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n", luas);

return 0;
}

File: Modul-02/python/PRAK205-2310817210009-Muhammad-Ibnu-Sina.py
import math

print("Input: ")
TINGGI, MIRING = map(float, input().split())

alas = int(math.sqrt(MIRING**2 - TINGGI**2))
tinggi = int(TINGGI)
keliling = int(TINGGI + MIRING + alas)
luas = int(0.5 * alas * tinggi)

print("\nOutput:\nAlas = {} cm".format(alas))
print("tinggi = {} cm".format(tinggi))
print("keliling = {} cm".format(keliling))
print("Luas = {} cm^2".format(luas))
